### PR TITLE
fixes the dependency of labeller example on plyr in issue #3083

### LINKED ISSUE #3083: not-run example in labeller() depends on unimported plyr
When @thomasp85 removed plyr, an example was changed so it doesn't run anymore. Since the example is in `\donttest{}`, the problem doesn't get flagged in automated testing.

The problematic line is here:
https://github.com/tidyverse/ggplot2/blob/f0da3d0cb8d6859976339bdf245f21b48991d640/R/labeller.r#L401

The formatted code in the example looks like this:
``` r
# In the following example, we rename the levels to the long form,
# then apply a wrap labeller to the columns to prevent cropped text
msleep$conservation2 <- revalue(msleep$conservation, conservation_status)
p3 <- ggplot(msleep, aes(x = sleep_total, y = awake)) + geom_point()
p3 +
  facet_grid(vore ~ conservation2,
    labeller = labeller(conservation2 = label_wrap_gen(10))
  )
```
I see three possible fixes. Not sure which one is best:

1. Add `plyr::` back in to that line of code. This would mean plyr has to be listed as suggested package.

2. We write `ggplot2:::revalue()` instead of `revalue()`.

3. We change or delete the example.

### PR DESCRIPTION
**Pull request by connorlewis**
Changed the use of revalue (from the plyr package) to base in the don't test example of labeller. 
-fixes #3083

### CODE REVIEW

**Comment by thomasp85** [comment]
Can I get you to update the docs with `devtools::document()`

**Comment by connorlewis** [comment]
Working on doing this now.

File: R/labeller.r [hunk 1]
```diff
@@ -398,7 +398,8 @@ as_labeller <- function(x, default = label_value, multi_line = TRUE) {
 #'
 #' # In the following example, we rename the levels to the long form,
 #' # then apply a wrap labeller to the columns to prevent cropped text
-#' msleep$conservation2 <- revalue(msleep$conservation, conservation_status)
+#' msleep$conservation2 <- conservation_status[match(msleep$conservation
+#'                                                  , names(conservation_status))]
```
[the remark this answers is not in the document]
Comment (clauswilke) [reply R/labeller.r]: Like so:
``` r
idx <- match(msleep$conservation, names(conservation_status))
msleep$conservation2 <- conservation_status[idx]
```

[the remark this answers is not in the document]
Comment (connorlewis) [reply R/labeller.r]: Changed, thanks for the assistance and opportunity to make the changes.

**Comment by clauswilke** [comment]
Thanks!

**Pull request merged**